### DISCUSSION TITLE
Comparing Results To Literature

### DISCUSSION THREAD

**Discussion by Lufter** [nikolasibalic/ARC-Alkali-Rydberg-Calculator#177] [Q&A] [answered]
Hi,
I am new to the ARC package, and while reading through the documentation on using ARC to calculate the dipole matrix element of the Cesium D2 Transition 6S_{1/2}-->6P_{3/2}, I noticed that the value was not what I expected. For [example](https://arc-alkali-rydberg-calculator.readthedocs.io/en/latest/Rydberg_atoms_a_primer_notebook.html#Matrix-Elements),

ARC is showing that the dipole matrix element is <j||er||j'> = 6.334 ea_0, while the [Steck's Cs D line note](https://steck.us/alkalidata/cesiumnumbers.1.6.pdf) is suggesting <j||er||j'> = 4.4786(40) ea0. I feel I might have misunderstood how to correctly read the results from ARC.

I also noticed that there is a potential typo in the documentation,
<img width="489" alt="Screenshot 2024-08-12 at 5 08 47 PM" src="https://github.com/user-attachments/assets/7c1098e0-775c-4981-9cda-790be2a1077a">

The sign of $q$ should be the opposite
<img width="627" alt="Screenshot 2024-08-12 at 5 09 24 PM" src="https://github.com/user-attachments/assets/edb16b5c-2208-4b83-872c-4d2d12d9c495">

**Comment by nikolasibalic** [comment, marked as answer]
Hi @Lufter 

Thank you for your question. Note that there are two definitions of reduced d.m.e. one that is symmetric in j, j', and the other one that is not. Notations for both are not universal so in each publication you need to double check.

In ARC we use  < || >  for symmetric, and (  ||  ) for asymmetric. In Steck notation is opositie. You can get both values from ARC calling [getReducedMatrixElementJ](https://arc-alkali-rydberg-calculator.readthedocs.io/en/latest/generated/arc.alkali_atom_functions.AlkaliAtom.getReducedMatrixElementJ.html) and [getReducedMatrixElementJ_asymmetric](https://arc-alkali-rydberg-calculator.readthedocs.io/en/latest/generated/arc.alkali_atom_functions.AlkaliAtom.getReducedMatrixElementJ_asymmetric.html).

I hope that helps.

**Reply by Lufter** [reply]
Thank you!
